WORKSPACE

```diff
@@ -106,7 +106,7 @@ graknlabs_grakn_core_artifacts()
 
 pip3_import(
     name = "graknlabs_client_python_pip",
-    requirements = "//:requirements.txt",
+    requirements = "//:requirements_dev.txt",
```

Comment:
When building the WORKSPACE, we want the _dev_ requirements that include `behave`. But when deploying the `pip` artifact, the _prod_ requirements (`requirements.txt`) is used, so it will not depend on `behave`.